jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -257,7 +257,11 @@ val ID_WRAPPERS =
                     listOf("project_report_configs\\.id", ".*\\.config_id")),
                 IdWrapper(
                     "ReportId",
-                    listOf("accelerator\\.reports\\.id", "accelerator\\..*\\.report_id")),
+                    listOf(
+                        "accelerator\\.reports\\.id",
+                        "accelerator\\..*\\.report_id",
+                        "funder\\.funder_reports\\.id",
```

Comment:
This line doesn't seem necessary any more (plus it refers to the old table name).